pkg/build/pipelines/go/build.yaml

```diff
@@ -71,6 +76,10 @@ pipeline:
           go get $dep
         done
         go mod tidy
+        # If vendor is specified, update the vendor directory
+        if [ "${{inputs.vendor}}" == "true" ]; then
```

Comment:
Why not
```suggestion
        ${{inputs.vendor}} && go mod vendor
```

Reply:
ooooooh using bool/string conversion to our advantage, nice :)